lms/djangoapps/verify_student/management/commands/manual_verifications.py

```diff
@@ -32,11 +32,24 @@ def add_arguments(self, parser):
             default=None,
             help='Path of the file to read email id from.',
             type=str,
-            required=True
+        )
+        parser.add_argument(
+            '--email',
+            default=None,
+            help='Single email to verify one user',
+            type=str,
         )
 
     def handle(self, *args, **options):
 
+        single_email = options['email']
+
+        if single_email:
+            successfully_verified =  self._add_user_to_manual_verification(single_email)
+            if successfully_verified is False:
+                log.error(f'Manual verification of {single_email} failed')
```

Comment:
This will only be used locally right? If it's going to be in prod, we probably don't want to log emails.

Reply:
Thats the idea, but there'd be nothing stopping someone except for maybe a stern warning. Should I reconsider the error?